Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -33,10 +33,10 @@ struct IntroViewControllerUX {
     static let CardTitleCustomize = NSLocalizedString("Customize", tableName: "Intro", comment: "See http://mzl.la/1T8gxwo")
     static let CardTitleShare = NSLocalizedString("Share", tableName: "Intro", comment: "See http://mzl.la/1if9ODp")
     static let CardTitleChoose = NSLocalizedString("Choose", tableName: "Intro", comment: "See http://mzl.la/1if9ODp")
+    static let CardTitleSync = NSLocalizedString("Sync your devices", tableName: "Intro", comment: "See http://mzl.la/1if9ODp")
```

Comment:
This will not work. We are past string freeze. The idea is to re-use the existing `Sync your Devices.` string. But now I can't find this string anymore. Weird I was so sure we had a string in the product that we could re-use for this.